### PR TITLE
Fix#97

### PR DESCRIPTION
**Pull request by dpwspoon**

### CODE REVIEW

File: driver/src/main/java/org/kaazing/k3po/driver/behavior/ScriptProgress.java [hunk 1]
```diff
@@ -58,21 +59,25 @@ public RegionInfo getScriptInfo() {
...
-            return expectedScript;
-        }
-        else {
-            StringBuilder builder = new StringBuilder();
-            processRegion(builder, scriptInfo, failureInfos);
-            return builder.toString();
+        if (observeredScript == null) {
+            if (failureInfos.size() == 0) {
+                // no failures
+                observeredScript =  expectedScript;
```
Comment (jfallows) [inline driver/src/main/java/org/kaazing/k3po/driver/behavior/ScriptProgress.java]: Extra space after =

File: lang/src/main/java/org/kaazing/k3po/lang/parser/ScriptParseStrategy.java [hunk 2]
```diff
@@ -2002,10 +2002,11 @@ public AstReadConfigNode visitReadHttpHeaderNode(ReadHttpHeaderNodeContext ctx)
 
             node = new AstReadConfigNode();
             node.setType(ctx.HttpMissingKeyword() != null ? "header missing" : "header");
-            node.setRegionInfo(asSequentialRegion(childInfos, ctx));
             node.setValue("name", value);
+            super.visitReadHttpHeaderNode(ctx);
+            node.setRegionInfo(asSequentialRegion(childInfos, ctx));
```
Comment (jfallows) [inline lang/src/main/java/org/kaazing/k3po/lang/parser/ScriptParseStrategy.java]: Nice catch @dpwspoon !

**Comment by jfallows** [comment]
Looks great, thanks!

**Comment by dpwspoon** [comment]
This includes fixes that ensure region info is properly added to child nodes in the parser, And this includes adding appveyor.yml since we have had windows specific bind problems.

@jfallows I believe this should be good to merge

File: lang/src/test/java/org/kaazing/k3po/lang/parser/ParseErrorsTest.java [hunk 3]
```diff
@@ -1,451 +0,0 @@
-/*
```
Comment (dpwspoon) [inline lang/src/test/java/org/kaazing/k3po/lang/parser/ParseErrorsTest.java]: Old parser tests I believe from older version of ANTLR, so I removed them

File: lang/src/test/java/org/kaazing/k3po/lang/parser/RobotScriptIRGeneratorTest.java [hunk 4]
```diff
@@ -1,630 +0,0 @@
-/*
```
Comment (dpwspoon) [inline lang/src/test/java/org/kaazing/k3po/lang/parser/RobotScriptIRGeneratorTest.java]: Old parser tests I believe from older version of ANTLR, so I removed them

**Pull request merged**